kubernetes-maven-plugin/plugin/src/main/java/org/eclipse/jkube/maven/plugin/mojo/build/ResourceMojo.java

```diff
@@ -249,7 +249,7 @@ private KubernetesList generateResources() throws IOException {
         return jkubeServiceHub.getResourceService().generateResources(getPlatformMode(), enricherManager, log);
     }
 
-    private ProcessorConfig extractEnricherConfig() throws IOException {
+    private ProcessorConfig extractEnricherConfig() {
```

Comment:
How is this change related to #2788 ?

Reply:
It is related to this PR that I worked on #2786

Reply:
Sorry, I meant why is this change here. 

Maybe if you rebase your branch against latest version of https://github.com/eclipse/jkube/tree/master this would not show up here.

Reply:
Noted, thank you